### PR TITLE
MBSA-486 Run fake queue in k8s

### PR DESCRIPTION
**Pull request by RuslanMingaliev**

### CODE REVIEW

File: subprojects/emcee/queue-fake-server/src/main/kotlin/com/avito/emcee/server/fake/Main.kt [hunk 1]
```diff
@@ -11,15 +11,19 @@ import io.ktor.server.engine.embeddedServer
 import io.ktor.server.netty.Netty
 import io.ktor.server.plugins.contentnegotiation.ContentNegotiation
 import io.ktor.server.response.respond
+import io.ktor.server.routing.get
 import io.ktor.server.routing.post
 import io.ktor.server.routing.routing
 
 public fun main() {
-    embeddedServer(Netty, port = 8080) {
+    embeddedServer(Netty, port = 41000) {
```
Comment (RuslanMingaliev) [inline subprojects/emcee/queue-fake-server/src/main/kotlin/com/avito/emcee/server/fake/Main.kt:19]: It doesn't matter, but the real queue runs on the 41000 port

File: subprojects/emcee/k8s/emcee.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    replicas: 1
+    template:
+        metadata:
+            labels:
+                app: emcee
+                service: emcee
+        spec:
+            containers:
+                -   name: emcee
+                    image: emcee-queue
```
Comment (sergioboi) [inline subprojects/emcee/k8s/emcee.yaml:24]: What is the image? How did you build it?

File: subprojects/emcee/k8s/emcee.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                    imagePullPolicy: IfNotPresent
+                    ports:
+                        -   containerPort: 41000
```
Comment (sergioboi) [inline subprojects/emcee/k8s/emcee.yaml:27]: Why do we need that?

File: subprojects/emcee/queue-fake-server/build.gradle.kts [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
     mainClass.set("com.avito.emcee.server.fake.MainKt")
 }
 
+val registry = project.getMandatoryStringProperty("avito.docker.registry", allowBlank = false)
+
+docker {
+    javaApplication {
+        baseImage.set("$registry/android/openjdk:11")
+        ports.set(setOf(41000))
+        images.set(setOf("emcee-queue:latest"))
```
Comment (sergioboi) [inline subprojects/emcee/queue-fake-server/build.gradle.kts:20]: I'm against of using `latest` but in current place It doesn't matter

File: subprojects/emcee/k8s/emcee.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (sergioboi) [inline subprojects/emcee/k8s/emcee.yaml:1]: I don't remember all details but could you add what we try to do via this yaml file
Could yaml have a comment?
As I see we create a `deployment` related to `service`. But do we need a `service`? Or we could do the same without `service` only with `deployment`?

**Review by sergioboi — APPROVED** [review]

File: subprojects/emcee/k8s/emcee.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        name: emcee
+        service: emcee
+spec:
+    type: NodePort
+    selector:
+        app: emcee
+    ports:
+        -   protocol: TCP
+            port: 41000
+            targetPort: 41000
```
Comment (beefon) [inline subprojects/emcee/k8s/emcee.yaml:44]: Emcee runs by default on default range of ports `[41000...41009]`, if it is important here.

File: subprojects/emcee/k8s/emcee.yaml (hunk 5 above)
Comment (RuslanMingaliev) [reply subprojects/emcee/k8s/emcee.yaml:1]: Add comments

File: subprojects/emcee/k8s/emcee.yaml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+    name: emcee
+    namespace: default
+    labels:
```
Comment (RuslanMingaliev) [inline subprojects/emcee/k8s/emcee.yaml:7]: Do we need these labels?

**Pull request closed without merging**